Providers/Extras/Scripts/logrotate.conf

```diff
@@ -7,3 +7,10 @@
 	/opt/omi/bin/service_control restart
     endscript
 }
+
+/var/opt/omi/log/dscdetailed.log {
+    rotate 5
+    sharedscripts
+    size 5M
+    compress
+}
```

Comment:
do we know why omi logs do not have size rotation ?

Reply:
I have no idea

---
In reply to: [140113473](https://github.com/Microsoft/PowerShell-DSC-for-Linux/pull/327#discussion_r140113473) [](ancestors = 140113473)

Reply:
ok. 

---
In reply to: [140364686](https://github.com/Microsoft/PowerShell-DSC-for-Linux/pull/327#discussion_r140364686) [](ancestors = 140364686,140113473)